Code/Framework/AzToolsFramework/AzToolsFramework/ComponentMode/ComponentModeSwitcher.cpp

```diff
@@ -147,12 +145,12 @@ namespace AzToolsFramework::ComponentModeFramework
         }
         else if (!newlyDeselectedEntityIds.empty())
         {
-            //CLear the switcher then add the components back if entities are still selected
+            // Clear the switcher then add the components back if entities are still selected
```

Comment:
uber nit: Just for consistency, the majority of viewport code has all comments in the implementation files (.cpps) not start with an upper case letter - so this would just be `clear the....` (we do always start with uppercase in the .h file though for API comments). Changing this will keep this file and others roundabout consistent (general style rule is follow what's there if in doubt 🙂). Thanks!